platform/viewer/tailwind.config.js

```diff
@@ -18,6 +18,11 @@ module.exports = {
     '../../node_modules/@ohif/extension-*/src/**/*.{js,jsx,css, ts,tsx}',
   ],
   theme: {
+    extend: {
+      spacing: {
+        '1.5': '0.375rem',
+      },
+    },
```

Comment:
which spacing is this? since tailwind version we use already has `p-1.5` and `w-1.5`

Reply:
I tried `px-1.5` without adding this and it did NOT work.

Reply:
quite bizarre.

https://tailwindcss.com/docs/padding

Maybe update our tailwind to 3.2.7

Reply:
Bizarre indeed. I will try.

Reply:
Updating/upgrading works. I suspect also that perhaps maybe when I first tried I had a typo or perhaps if tailwind does some caching, I had a stale cache. Regardless, I think we are good to upgrade and remove this extension I added. Thank you for having me double check this.

Reply:
Oh great news